src/components/pages/About/About.jsx

```diff
@@ -1,12 +1,14 @@
 import React from "react";
 import Clients from "../../About/Clients";
 import AboutDetails from "../../About/AboutDetails";
+import Directors from "../../About/Directors";
 //import styles from "./About.module.css";
```

Comment:
I think finally this can be removed as we no longer need a styling for the entire page